main/tests/server/src/com/google/refine/importers/RdfTripleImporterTests.java

```diff
@@ -64,115 +64,6 @@ public void setUp() {
         SUT = new RdfTripleImporter();
         JSONUtilities.safePut(options, "base-url", "http://rdf.freebase.com");
     }
-
-    @Test(enabled = false)
-    public void canParseSingleLineTriple() {
-        String sampleRdf = "<http://rdf.freebase.com/ns/en.bob_dylan> <http://rdf.freebase.com/ns/music.artist.album> <http://rdf.freebase.com/ns/en.blood_on_the_tracks>.";
-        StringReader reader = new StringReader(sampleRdf);
-
-        try {
-            parseOneFile(SUT, reader);
-        } catch (Exception e) {
-            Assert.fail();
-        }
-
-        Assert.assertEquals(project.columnModel.columns.size(), 2);
-        Assert.assertEquals(project.columnModel.columns.get(0).getName(), "subject");
-        Assert.assertEquals(project.columnModel.columns.get(1).getName(), "http://rdf.freebase.com/ns/music.artist.album");
-        Assert.assertEquals(project.rows.size(), 1);
-        Assert.assertEquals(project.rows.get(0).cells.size(), 2);
-        Assert.assertEquals(project.rows.get(0).cells.get(0).value, "http://rdf.freebase.com/ns/en.bob_dylan");
-        Assert.assertEquals(project.rows.get(0).cells.get(1).value, "http://rdf.freebase.com/ns/en.blood_on_the_tracks");
-    }
-
-    @Test
-    public void canParseMultiLineTriple() throws UnsupportedEncodingException {
-        String sampleRdf = "<http://rdf.freebase.com/ns/en.bob_dylan> <http://rdf.freebase.com/ns/music.artist.album> <http://rdf.freebase.com/ns/en.blood_on_the_tracks>.\n"
-                +
-                "<http://rdf.freebase.com/ns/en.bob_dylan> <http://rdf.freebase.com/ns/music.artist.album> <http://rdf.freebase.com/ns/en.under_the_red_sky>.\n"
-                +
-                "<http://rdf.freebase.com/ns/en.bob_dylan> <http://rdf.freebase.com/ns/music.artist.album> <http://rdf.freebase.com/ns/en.bringing_it_all_back_home>.";
```

Comment:
Although those triples refer to Freebase, this tests the RDF importer, which is agnostic about which domains are used in the URIs it parses, so I would expect that this test should be kept.

For the sake of making clear that this is unrelated to Freebase integration, we could replace `rdf.freebase.com` by any other domain.

Reply:
Alright, I added back the tests and changed the URLs.